torch/csrc/utils/qengines.cpp

```diff
@@ -28,6 +28,7 @@ void initializeQEngines() {
     throw python_error();
   }
 
+  addQEngine(at::kNoQEngine, "no_qengine", torch_module);
```

Comment:
sigh, we should really change it to strings, it's getting really polluting to the namespace